print-mptcp.c

```diff
@@ -66,6 +66,18 @@ struct mptcp_option {
 
 #define MPTCP_OPT_SUBTYPE(sub_etc)      (((sub_etc) >> 4) & 0xF)
 
+static const struct tok mp_capable_flags[] = {
+        { 0x80, "A" },
```

Comment:
Perhaps `MP_CAPABLE_A`, since it exists now?

Reply:
Just to make sure you mean 0x80 -> MP_CAPABLE_A ?

Reply:
Yes.

Reply:
Yhea it should be more clear.